lib/rapis/converter.rb

```diff
@@ -50,29 +52,70 @@ def to_dsl_exclude_key
     def to_dsl_key_conv
       proc do |k|
         k = k.to_s
-        if k =~ %r{^\/}
-          proc do |v, nested|
-            if nested
-              "path #{k.inspect} #{v}"
-            else
-              "path #{k.inspect}, #{v}"
-            end
-          end
-        elsif k =~ /^\d{3}$/
-          proc do |v, nested|
-            if nested
-              "code #{k} #{v}"
-            else
-              "code #{k}, #{v}"
-            end
-          end
+        case k
+        when %r{^\/}
+          item_key_proc k
+        when /^\d{3}$/
+          code_key_proc k
+        when 'parameters', 'api_key'
+          parameters_key_proc k
         else
           CHANGE_SETS.each { |f, t| k = k.gsub(f, t) }
           k
         end
       end
     end
 
+    def item_key_proc(k)
+      proc do |v, nested|
+        if nested
+          "item #{k.inspect} #{v}"
+        else
+          "item #{k.inspect}, #{v}"
+        end
+      end
+    end
+
+    def code_key_proc(k)
+      proc do |v, nested|
+        if nested
+          "code #{k.to_s.inspect} #{v}"
+        else
+          "code #{k.to_s.inspect}, #{v}"
+        end
+      end
+    end
+
+    def parameters_key_proc(k)
+      proc do |v, nested|
+        if v =~ /\n(\s+)/
+          indent = $1
```

Comment:
__[rubocop]__
* Avoid the use of Perl-style backrefs. [:ref](https://github.com/bbatsov/ruby-style-guide#no-perl-regexp-last-matchers)